README.md

```diff
@@ -483,6 +484,22 @@ The data format for donut, pie, and radial bar are the simplest. They are
 just any single value of type Numeric.
 
 
+## Options (Reference)
+ApexCharts.rb supports [all options from apexcharts.js](https://apexcharts.com/docs/options/). They should be written in snake_case instead of camelCase.
+
+These options can be passed to any Chart helper like `<%= line_chart(series, options) %>`:
```

Comment:
- "Chart" should be lowercase.
- please come up with other example of options (the same options can be boring) and put it before this sentence.
- don't forget to end this sentence with a dot.

Reply:
Updated